pom.xml

```diff
@@ -128,7 +128,7 @@
                         <configuration>
                             <referenceVersion>2.22ea3</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>99</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>95.1</binaryCompatibilityPercentageRequired>
```

Comment:
ugg !

Reply:
Yepp. It is because some protected classes got protected constructors and because Jvm is a proper utility class now like OS and not an enum.